packages/runner/src/lib/event-manager.js

```diff
@@ -197,40 +209,44 @@ const eventManager = {
   initialize ($autIframe, config) {
     performance.mark('initialize-start')
 
-    Cypress.initialize($autIframe)
-
-    // get the current runnable in case we reran mid-test due to a visit
-    // to a new domain
-    ws.emit('get:existing:run:state', (state = {}) => {
-      const runnables = Cypress.normalizeAll(state.tests)
-      const run = () => {
-        performance.mark('initialize-end')
-        performance.measure('initialize', 'initialize-start', 'initialize-end')
-        this._runDriver(state)
-      }
-
-      reporterBus.emit('runnables:ready', runnables)
-
-      if (state.numLogs) {
-        Cypress.setNumLogs(state.numLogs)
-      }
-
-      if (state.startTime) {
-        Cypress.setStartTime(state.startTime)
-      }
-
-      if (state.currentId) {
-        // if we have a currentId it means
-        // we need to tell the Cypress to skip
-        // ahead to that test
-        Cypress.resumeAtTest(state.currentId, state.emissions)
-      }
-
-      if (config.isTextTerminal && !state.currentId) {
-        ws.emit('set:runnables', runnables, run)
-      } else {
-        run()
-      }
+    return Cypress.initialize({
+      $autIframe,
+      onSpecReady: () => {
+        // get the current runnable in case we reran mid-test due to a visit
+        // to a new domain
+        ws.emit('get:existing:run:state', (state = {}) => {
+          const runnables = Cypress.normalizeAll(state.tests)
+          const run = () => {
+            performance.mark('initialize-end')
+            performance.measure('initialize', 'initialize-start', 'initialize-end')
```

Comment:
did we mean to keep this in production?

Reply:
That wasn't introduced here. The indentation changed, that's the reason it's in the diff.